### ISSUE TITLE
508 compliance -- scrolling attribute

### ISSUE THREAD

**Issue by mdobbrow** [davidjbradshaw/iframe-resizer#544]
Hi - 

we're using the iframe-resizer on our school district website - works great - thanks!

One issue we've found though, is it fails 508 Accessibility test (a requirement for all gov't and school websites); this is due we believe to the scrolling="no" attribute added to the iframe. Section 508 requires that display-style attributes are added via css classes. Would it be hard to make a version that uses a css class to set scrolling?

Thanks -- any help greatly appreciated!
Mark

**Comment by davidjbradshaw** [comment]
Hi,

You can change what value it sets their by a config option.

https://github.com/davidjbradshaw/iframe-resizer#scrolling

and their is nothing stopping setting a matching value in CSS.

It is set in code as older browsers don't support setting it in CSS. I a reasonable amount of experience of 508 and w3c WCAG 2.0 and I have not seen that requirement before. Can you let me know where your referencing it from?

D.
